tests/TestCase/Http/Cookie/CookieTest.php

```diff
@@ -510,4 +511,17 @@ public function testCreateFromHeaderString()
         // https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-4.1
         $this->assertNull($result->getSameSite());
     }
+
+    public function testDefaults()
+    {
+        Cookie::setDefaults(['path' => '/cakephp', 'expires' => time()]);
```

Comment:
We should have a test for invalid same site and expires values.

Reply:
Convenience of array means losing benefit of type hinting :sob:. I need array shapes :smile:.

Reply:
I added checks and tests :)